os/include/tinyara/net/if/wifi.h

```diff
@@ -70,6 +70,17 @@ typedef struct {
 	trwifi_ap_crypto_type_e ap_crypto_type;		 /**<  @ref trwifi_ap_crypto_type	   */
 } trwifi_ap_config_s;
 
+typedef struct {
+	unsigned char scan_mode;					/**<	0:full, 1:partial											*/
+	unsigned char scan_opt;						/**<	0:active, 1:passive in all channel, 2:force active		*/
+	unsigned char *channel_2g;					/**<	2g channel list											*/
+	unsigned char channel_2g_len;				/**<	2g channel length											*/
+	unsigned char *channel_5g;					/**<	5g channel list											*/
+	unsigned char channel_5g_len;				/**<	5g channel length											*/
```

Comment:
please maintain the same tab space for comments

Reply:
Done